processed 6 tasks

init:
A: object(0,0)

task 1 'publish'. lines 7-28:
created: object(1,0)
mutated: object(0,1)
gas summary: computation_cost: 1000000, storage_cost: 5821600,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2 'run'. lines 30-30:
Error: Transaction Effects Status: Insufficient Gas.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: InsufficientGas, source: Some(VMError { major_status: OUT_OF_GAS, sub_status: None, message: None, exec_state: Some(ExecutionState { stack_trace: [(Some(ModuleId { address: sui, name: Identifier("object") }), FunctionDefinitionIndex(10), 1), (Some(ModuleId { address: a, name: Identifier("m") }), FunctionDefinitionIndex(0), 8)] }), location: Module(ModuleId { address: sui, name: Identifier("tx_context") }), indices: [], offsets: [(FunctionDefinitionIndex(4), 0)] }), command: Some(0) } }

task 3 'run'. lines 32-32:
Error: Transaction Effects Status: Insufficient Gas.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: InsufficientGas, source: Some(VMError { major_status: OUT_OF_GAS, sub_status: None, message: None, exec_state: Some(ExecutionState { stack_trace: [(Some(ModuleId { address: sui, name: Identifier("object") }), FunctionDefinitionIndex(10), 1), (Some(ModuleId { address: a, name: Identifier("m") }), FunctionDefinitionIndex(0), 8)] }), location: Module(ModuleId { address: sui, name: Identifier("tx_context") }), indices: [], offsets: [(FunctionDefinitionIndex(4), 0)] }), command: Some(0) } }

task 4 'run'. lines 34-34:
Error: Transaction Effects Status: Insufficient Gas.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: InsufficientGas, source: Some(VMError { major_status: OUT_OF_GAS, sub_status: None, message: None, exec_state: Some(ExecutionState { stack_trace: [(Some(ModuleId { address: sui, name: Identifier("object") }), FunctionDefinitionIndex(10), 1), (Some(ModuleId { address: a, name: Identifier("m") }), FunctionDefinitionIndex(0), 8)] }), location: Module(ModuleId { address: sui, name: Identifier("tx_context") }), indices: [], offsets: [(FunctionDefinitionIndex(4), 0)] }), command: Some(0) } }

task 5 'run'. lines 36-36:
Error: Transaction Effects Status: Move Primitive Runtime Error. Location: sui::dynamic_field::has_child_object (function index 14) at offset 0. Arithmetic error, stack overflow, max value depth, etc.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MovePrimitiveRuntimeError(MoveLocationOpt(Some(MoveLocation { module: ModuleId { address: sui, name: Identifier("dynamic_field") }, function: 14, instruction: 0, function_name: Some("has_child_object") }))), source: Some(VMError { major_status: MEMORY_LIMIT_EXCEEDED, sub_status: Some(5), message: Some("Object runtime cached objects limit (1000 entries) reached"), exec_state: None, location: Module(ModuleId { address: sui, name: Identifier("dynamic_field") }), indices: [], offsets: [(FunctionDefinitionIndex(14), 0)] }), command: Some(0) } }
